apps/shopify/src/skuResolvers.js

```diff
@@ -69,25 +76,29 @@ export const fetchCollectionPreviews = async (skus, config) => {
 
 /**
  * Fetches the product previews for the products selected by the user.
- *
- * Note: currently there is no way to cover the edge case where the user
- *       would have more than 250 products selected. In such a case their
- *       selection would be cut off after product no. 250.
  */
 export const fetchProductPreviews = async (skus, config) => {
   if (!skus.length) {
     return [];
   }
-
   const validIds = filterAndDecodeValidIds(skus, 'Product');
   const shopifyClient = await makeShopifyClient(config);
-  const response = await shopifyClient.product.fetchMultiple(validIds);
+
+  const requests = [];
+  for (let i = 0; i < validIds.length; i += SHOPIFY_ENTITY_LIMIT) {
+    requests.push(
+      shopifyClient.product.fetchMultiple(validIds.slice(i, i + (SHOPIFY_ENTITY_LIMIT - 1)))
+    );
+  }
+  const response = (await Promise.all(requests)).reduce((acc, curr) => [...acc, ...curr], []);
```

Comment:
If you change `response` to a `let` above, then you can reassign without having to do `const` again.

Reply:
I prefer to write code with solely consts, I find it's easier for developers to read code that's declarative vs imperative. Leads to less cognitive load when trying to keep track of what's happening.

I'm down to discuss this though!

Reply:
Hmmmm, I take your point, as long as we aren't re-declaring `const` values then we're good.